release_circleci.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
Why the split between `release_circleci.sh` and `release_local.sh` ?

Reply:
@moshloop As discussed, keeping one file release.sh only.